core/app/mailers/spree/promotion_code_batch_mailer.rb

```diff
@@ -0,0 +1,31 @@
+module Spree
+  class PromotionCodeBatchMailer < ApplicationMailer
+    def promotion_code_batch_finished(promotion_code_batch)
+      mail(
+        to: promotion_code_batch.email,
+        body: Spree.t(
```

Comment:
I don't like to hard code the body in here. Please let's use mailer views as for all the other mailers.